maat-court-data-api/src/main/java/gov/uk/courtdata/entity/PassportAssessmentEvidenceEntity.java

```diff
@@ -0,0 +1,78 @@
+package gov.uk.courtdata.entity;
+
+import com.fasterxml.jackson.annotation.JsonBackReference;
+import com.fasterxml.jackson.annotation.JsonManagedReference;
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import java.time.LocalDateTime;
+
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@Entity
+@Table(name = "PASSPORT_ASS_EVIDENCE", schema = "TOGDATA", indexes = {
+        @Index(name = "PAAS_UK", columnList = "PAAS_ID, INEV_EVIDENCE, REMOVED_DATE, APPL_ID, OTHER_TEXT", unique = true)
+})
+public class PassportAssessmentEvidenceEntity {
+    @Id
+    @SequenceGenerator(name = "passport_ass_income_evidence_gen_seq", sequenceName = "S_GENERAL_SEQUENCE", allocationSize = 1, schema = "TOGDATA")
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "passport_ass_income_evidence_gen_seq")
+    @Column(name = "ID", nullable = false)
+    private Integer id;
+
+    @JsonBackReference
+    @ManyToOne(optional = false)
+    @JoinColumn(name = "PAAS_ID", nullable = false)
+    private PassportAssessmentEntity passportAssessment;
+
+    @Column(name = "DATE_RECEIVED")
+    private LocalDateTime dateReceived;
```

Comment:
I noticed on the table this and REMOVED_DATE are just DATE type, does this need to be defined as just LocalDate type in the code here?

Reply:
That is correct, we are keeping this inline with the FIN_ASS_INCOME_EVIDENCE

Reply:
Ok that makes sense, happy to resolve as we need to keep them both in line to make the logic in orchestrator simpler.